components/globals/ReleaseNote.vue

```diff
@@ -6,7 +6,7 @@
       <h2 :id="id" class="mb-1">
         {{ title }}
       </h2>
-      <p class="text-14 text-font-alt3">{{ date }}</p>
+      <time class="text-14 text-font-alt3">{{ date }}</time>
```

Comment:
Can you add the date in the time class as well. `<time datetime="2018-07-07">`

Reply:
```suggestion
      <time :datetime="date" class="text-14 text-font-alt3">{{ date }}</time>
```

Not sure if this works in Vue.